### ISSUE TITLE
Destiny.Definitions.DestinyVendorDefinition > Missing property in documentation

### ISSUE THREAD

**Issue by thisbarker** [Bungie-net/api#73]
Request: https://www.bungie.net/Platform/Destiny2/{membershipType}/Profile/{destinyMembershipId}/?components={componentList}

Destiny.Definitions.DestinyVendorDefinition.**canAcquire**
- Response.profileKiosks.data.kioskItems.622587395[0].canAcquire

**Comment by vthornheart-bng** [comment]
Hmm - I see the issue here, the documentation is only showing "array" as the dictionary contents, rather than mentioning that it's an array of DestinyKioskItem.  That's very confusing, and I can see where it'd seem like it's actually returning DestinyVendorDefinition as a result!  I'll have to fix that, good catch!

**Comment by vthornheart-bng** [comment]
Yeah, turns out there's two places where we return List<X> as the values of a Dictionary, and both of those places just say "array".  Definitely a documentation generation issue, possibly a spec generation issue... will look into it soon.  In the meantime, https://bungie-net.github.io/#/components/schemas/Destiny.Components.Kiosks.DestinyKioskItem will take you to the actual definition meant to be seen there.  The values of that dictionary are lists of DestinyKioskItem - one per kiosk, keyed by the kiosk vendor's hash identifier!

**Comment by bhollis** [comment]
This is possibly obsolete now that kiosks are no longer used in the API (is that correct? I don't see them returned anymore - might want to even remove them from the API docs).
